packages/apps/tools/vitest.config.ts

```diff
@@ -12,10 +12,10 @@ export default mergeConfig(
       threads: false,
       coverage: {
         provider: 'v8',
-        lines: 70,
-        functions: 40,
+        lines: 73.76,
```

Comment:
What are these values?

Reply:
These are the required coverage thresholds, meaning if your unit tests have less then the required thresholds: You won't be able to merge. Thresholds have been defined globally at 90% for lines, functions branches and statements. 


The tools dApp currently does not meet these, which is why we're overriding these values for this specific package with lower values. The parameter `thresholdAutoUpdate` will automatically increase these values as you write more unit tests. Once the tools dApp meets the required thresholds of 90%, these overrides can be removed.

Reply:
Ah super, thanks for the explanation 👌